.github/workflows/lint.yml

```diff
@@ -48,7 +48,7 @@ jobs:
             ${{ runner.os }}-${{ matrix.node-version }}-nodemodules-
 
       - name: Install packages
-        run: yarn
```

Comment:
what does `--immutable` do?

Reply:
It will fail in case `yarn install` produces diffs that are not already in the lockfile, hence ensuring the lockfile is always up to date.

Reply:
ah, that's great!